src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+function getAvarageGrade(grades: number[]): number {
+  return grades.reduce((sum, n) => sum + n) / grades.length;
+}
+
+export function sortStudents(students: Student[],
+  sortBy: SortType, order: SortOrder): Student[] {
```

Comment:
```suggestion
export function sortStudents(
  students: Student[],
  sortBy: SortType,
  order: SortOrder
): Student[] {
```
better formatting :arrow_up: 
just FYI